src/js/index.js

```diff
@@ -186,6 +186,9 @@ export const hpe = deepFreeze({
       border: {
         radius: '4px',
       },
+      extend: ({ theme }) =>
+        `margin-top: ${theme.global.edgeSize.xsmall}; 
```

Comment:
For future reference, `theme.global.edgeSize.xsmall` = `6px` which aligns with what is shown in designs.